IM_calculation/IM/Burks_Baker_2013_elastic_inelastic.py

```diff
@@ -41,8 +41,19 @@
 
 
 def Bilinear_Newmark_withTH(
-    period: np.ndarray, z: float, dy: float, alpha: float, ag: np.ndarray, dt: float
+    period: np.ndarray,
+    z: float,
+    dy: float,
+    alpha: float,
+    ag: np.ndarray,
+    Dtg: float,
+    Dt: float,
 ):
+    # Analysis time step
+    # If Dt is too high in relation to the period, adjust it to ensure numerical stability
+    if Dt is None or Dt / np.min(period) > 0.551:
```

Comment:
Dt should never be passed in as None. It should either have a default value of None or this test case should be removed